src/components/AvatarGroup/__test__/avatarGroup.spec.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AvatarGroup from '../';
+import RenderIf from '../../RenderIf';
+
+jest.mock('../../RenderIf', () => () =>
+    // eslint-disable-next-line no-shadow
+    RenderIf => props => <RenderIf {...props} />,
+);
```

Comment:
Mocks are usually used with helper functions, in the case of other components is not necessary to mock them.